packages/shared/lib/services/connection.service.ts

```diff
@@ -2054,7 +2054,7 @@ class ConnectionService {
     async countMetric(): Promise<
         Result<
             {
-                accountId: number;
+                account_id: number;
```

Comment:
pg doesn't like camel case and transform field name to lowercase

Reply:
it works with quotes